pgmpy/base/DirectedGraph.py

```diff
@@ -315,3 +315,22 @@ def get_roots(self):
         ['A', 'E']
         """
         return [node for node, in_degree in self.in_degree().items() if in_degree == 0]
+    
+    def get_children(self, node):
+        """
+        Returns a list of children of node.
+        Throws an error if the node is not present in the graph.
+        Parameters
```

Comment:
@harishkashyap Need an empty line before this so that our docs get properly rendered.